cartridges/int_adyen_SFRA/cartridge/controllers/Adyen.js

```diff
@@ -41,7 +41,7 @@ server.post('AuthorizeWithForm', server.middleware.https, function (req, res, ne
         };
         var result = adyenCheckout.doPaymentDetailsCall(jsonRequest);
         Transaction.wrap(function () {
-            paymentInstrument.custom.adyenPaymentData = "";
+            paymentInstrument.custom.adyenPaymentData = null;
```

Comment:
Is this indeed different?

`paymentInstrument.custom.adyenPaymentData = "";` vs.
`paymentInstrument.custom.adyenPaymentData = null;`

Reply:
No, it is not different, but nicer to set it to null